templates/zerver/find_account.html

```diff
@@ -42,8 +22,8 @@ <h1 class="get-started">{{ _("Find your Zulip accounts") }}</h1>
                     you can <a href="/help/change-your-password">reset it</a>.
                     {% endtrans %}
                 </p>
-                <form class="form-inline" id="find_account" name="email_form"
-                  action="{{ current_url() }}" method="post">
+                <form class="form-inline" id="find_account" name="email_form" action="{{ current_url() }}"
+                  method="post">
```

Comment:
Let's not make these changes here for consistency. You can git grep "<form" to see that we follow the existing structure at other places.